public/static/locales/en/cognito-errors.json

```diff
@@ -18,7 +18,7 @@
   },
   "UsernameOrPasswordIncorrect": {
     "title": "The email address or password you entered is incorrect.",
-    "description": "The email address or password you entered is incorrect or the account does not exist.",
+    "description": "The email address or password you entered is incorrect or the .",
```

Comment:
Looks like this sentence is incomplete?

Reply:
Let's ignore this one, it was an old branch / set of changes  ... plus the other change was in your most recent PR. :)